WTWSMS/events/ze_lord_spiritual_events.txt

```diff
@@ -29,7 +29,6 @@ character_event = {
 			independent = yes
 			higher_tier_than = DUKE
 		}
-		proselyte_religion_trigger = yes
```

Comment:
We can make a duplicate of the actual Vanilla version and subject it to that rule.

Although, I do think our duplicate should not be able to convert Zealous characters, and we should keep the positive thirty five modifier.